.github/workflows/build_iso.yml

```diff
@@ -1,62 +1,140 @@
 name: Build Bluefin ISOs
 on:
   workflow_dispatch:
-  pull_request:
-    branches:
-      - main
-    paths:
-      - '.github/workflows/build_iso.yml'
-
-env:
-  IMAGE_REGISTRY: ghcr.io/${{ github.repository_owner }}
+  workflow_call:
```

Comment:
IMO controlling stuff like whether to push to R2 or GHA, and the channel (stable / testing) should be controlled in the calling workflow, but this is a nit

Reply:
Would be better, but we currently manually call `build_iso.yml` on it's own for uploading ISOs, so it needs to operate independently of the other workflow.